client/blocks/eligibility-warnings/hold-list.tsx

```diff
@@ -28,11 +28,17 @@ function getHoldMessages( translate: LocalizeProps[ 'translate' ] ) {
 			supportUrl: null,
 		},
 		SITE_PRIVATE: {
-			title: translate( 'Public site needed' ),
-			description: translate(
-				'Change your site\'s Privacy settings to "Public" or "Hidden" (not "Private.")'
-			),
-			supportUrl: localizeUrl( 'https://en.support.wordpress.com/settings/privacy-settings/' ),
+			title: isUnlaunched ? translate( 'Launch your site' ) : translate( 'Make site public' ),
+			description: isUnlaunched
+				? translate(
+						"This creates the basic layer the fancier stuff is built on. Don't worry, you can share this with everyone only when you're ready."
+				  )
+				: translate(
+						"Change's your site's Privacy settings to \"Hidden\". Don't worry, you can share this with everyone only when you're ready."
```

Comment:
The header and description are contradictory: Says "make it public" then description says that was change settings to "hidden" - meanwhile these are show as two separate options in the support page that is linked. 

Need to clarify the second requirement point so user knows precisely which to choose.

Reply:
You're right that is confusing 😕 This will be a lot easier one search engine indexing is a checkbox under Public paObgF-MY-p2

Reply:
I've updated it and focused more on the visibility aspect instead, and copied some of the wording from the settings view.
<img width="730" alt="Screenshot 2019-12-12 at 4 18 11 PM" src="https://user-images.githubusercontent.com/1500769/70679990-a511d600-1cfb-11ea-98b8-a6acdd0a3e51.png">

Reply:
Thank you! That's definitely better, but still little unclear on whether they will need to take an action or this will happen automatically when they upgrade & install something. If they do need to take action, will they be prompted later?

Reply:
Making the site visible happens automatically when clicking "Upgrade and continue". Maybe it's confusing the heading says "you'll need a couple things" when really _you_ don't need to do anything (except confirm).

Reply:
How about changing `Make site visible` to `A public site`?

Reply:
also cc @ianstewart

Reply:
One issue I see is that we say that the user will need things. Then the items are actions. We should make both items things.

> ### To install plugins, you'll need a couple of things:
> 
> **A Business plan**
> …
> 
> **A Public site**
> …

Reply:
The root PR now says "You'll need to:" so the phrasing here would work